projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -229,31 +242,6 @@ It's recommended to inject all dependencies as effect function arguments for eas
 
 ## Registering Root Effects
```

Comment:
We can consolidate the `Registering Root Effects` and `Registering Feature Effects` into one, as it's the same for the standalone API.
Can you give this a try or do you'd like help with this?
E.g. the text "Registering effects can also be done using the standalone APIs..." also needs to be updated.

Reply:
@timdeschryver i did it, but not sure if its correct.